### DISCUSSION TITLE
`[testenv]` `install_command` is also used for provision environment?

### DISCUSSION THREAD

**Discussion by oakkitten** [tox-dev/tox#2427] [General]
If I specify a high `minversion` or `requires`, a provision environment is created. Apparently, it is using `install_command` from `[testenv]` section. Is this right? I thought that `testenv` means test environment and will not be used for anything else.

If this is correct behavior, perhaps document it? Also document the way to make provision environment not use test environment. Apparently, this works:

```
[testenv:.tox]
install_command = python -m pip install {opts} {packages}

[testenv]
install_command = your_custom_command
```

**Comment by jugmac00** [comment]
Thanks! Could you please create an issue?

I could reproduce the issue both on tox version 3 and the upcoming tox version 4.

**Reply by oakkitten** [reply]
Created #2428 along with a few other notes about `install_command`.

**Reply by gaborbernat** [reply]
This is by design and the issue is in our documentation.

**Comment by gaborbernat** [comment]
Closing — an issue was requested but never created, and this is from 2022. The provision environment behavior is documented in tox 4. Feel free to open an issue if this is still relevant.
